src/main/java/com/rocketden/main/exception/TimerError.java

```diff
@@ -0,0 +1,24 @@
+package com.rocketden.main.exception;
+
+import com.rocketden.main.exception.api.ApiError;
+import com.rocketden.main.exception.api.ApiErrorResponse;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public enum TimerError implements ApiError {
+
+    NULL_SETTING(HttpStatus.BAD_REQUEST, "The game, associated game timer, room, room ID, and socket service must not be null.");
```

Comment:
When does this error get triggered? Like would it be a user error or an "us" error (i.e. like a 500 internal server mistake perhaps)?

Reply:
I think it would be an "us" error only -- I'm open to removing this if we want, what do you think?